src/fundus/publishers/de/sz.py

```diff
@@ -42,3 +48,9 @@ def title(self) -> Optional[str]:
         @attribute
         def topics(self) -> List[str]:
             return generic_topic_parsing(self.precomputed.ld.bf_search("keywords"))
+
+        @attribute(validate=False)
+        def free_access(self) -> bool:
+            if self.precomputed.ld.bf_search("isAccessibleForFree") is None:
+                return True
+            return not self.precomputed.ld.bf_search("isAccessibleForFree") == "False"
```

Comment:
Since there is an ongoing debate about weather we add this as a core attribute or not i would rather do this all at once. Maybe an issue about this topic would help.